network_manager/alembic/versions/242f91dc05ae_create_nodes_table.py

```diff
@@ -0,0 +1,45 @@
+"""Create Nodes Table
+
+Revision ID: 242f91dc05ae
+Revises:
+Create Date: 2023-04-21 08:47:13.216912
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = "242f91dc05ae"
+down_revision = None
+branch_labels = None
+depends_on = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
```

Comment:
I updated models.py and reran the migration so that:

- most columns should not be nullable (see below)
- unique constraints should be reflected in the migration
- column names should be same as the values in the node info json file